### ISSUE TITLE
Farmer Crashes On Receiving Nether Wart

### ISSUE THREAD

**Issue by aszecsei** [MelnCat/ActuallySubtractions#5]
When a farmer tries to re-plant nether wart, I get the following crash:

```
java.lang.ClassCastException: class net.minecraft.world.item.ItemNameBlockItem cannot be cast to class net.minecraftforge.common.IPlantable (net.minecraft.world.item.ItemNameBlockItem is in module minecraft@1.20.1 of loader 'TRANSFORMER' @69abeb14; net.minecraftforge.common.IPlantable is in module forge@47.3.0 of loader 'TRANSFORMER' @69abeb14)
	at TRANSFORMER/actuallyadditions@1.2.14.4+mc1.20.1/de.ellpeck.actuallyadditions.mod.misc.apiimpl.farmer.NetherWartFarmerBehavior.tryPlantSeed(NetherWartFarmerBehavior.java:41)
	at TRANSFORMER/actuallyadditions@1.2.14.4+mc1.20.1/de.ellpeck.actuallyadditions.mod.tile.TileEntityFarmer.checkBehaviors(TileEntityFarmer.java:156)
	at TRANSFORMER/actuallyadditions@1.2.14.4+mc1.20.1/de.ellpeck.actuallyadditions.mod.tile.TileEntityFarmer.serverTick(TileEntityFarmer.java:111)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.world.level.chunk.LevelChunk$BoundTickingBlockEntity.mixinextras$bridge$m_155252_$11(LevelChunk.java)
	at TRANSFORMER/neruina@2.1.2/com.bawnorton.neruina.handler.TickHandler.safelyTickBlockEntity(TickHandler.java:169)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.world.level.chunk.LevelChunk$BoundTickingBlockEntity.wrapOperation$epn000$catchTickingBlockEntity$notTheCauseOfTickLag(LevelChunk.java:1315)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.world.level.chunk.LevelChunk$BoundTickingBlockEntity.m_142224_(LevelChunk.java:689)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.world.level.chunk.LevelChunk$RebindableTickingBlockEntityWrapper.m_142224_(LevelChunk.java:782)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.world.level.Level.m_46463_(Level.java:468)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.level.ServerLevel.m_8793_(ServerLevel.java:351)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:893)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:814)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.client.server.IntegratedServer.m_5705_(IntegratedServer.java:89)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:661)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:251)
	at java.base/java.lang.Thread.run(Unknown Source)
	```

**Comment by MelnCat** [comment]
I'll look into it.

**Comment by MelnCat** [comment]
Fixed in 1.2.14.5.
